packages/cms-lib/lib/environment.js

```diff
@@ -0,0 +1,34 @@
+const { ENVIRONMENTS } = require('./constants');
+
+/**
+ * Returns environment constant for QA and PROD or optional masked value for PROD
+ * @param {string} env Environment string, can be any case
+ * @param {(boolean|object)} shouldMaskProduction Returning alternate value for PROD
+ *    Can be used to hide env value in the config. Boolean can be used to simply mask 'prod' with ''. The default
+ *    is not to modify the returned value for production, and instead return 'prod'.
+ * @param {any} shouldMaskProduction.maskedProductionValue Alternate value to return in place of PROD
+ */
+const getValidEnv = (env, shouldMaskProduction) => {
```

Comment:
This function introduces a "boolean trap" that makes reading code using this function difficult. It also feels like all this environment stuff is more complex than it needs to be. Is there a reason for all the complexity that we could fix through adjustments elsewhere?